README.md

```diff
@@ -63,7 +63,7 @@ Some of the major differences between this library and Numenta's extension libra
     - Version 2.7  We recommend you use the latest 2.7 version where possible. But the system version should be fine. (The extension library for Python 2.7 not supported on Windows.)
     - Version 3.4+  The Nupic Python repository will need to be upgraded as well before this will be useful.
   Be sure that your Python executable is in the Path environment variable. The Python that is in your default path is the one
-  that will determine which version of Python the extension library will be built for.
+  that will determine which version of Python the extension library will be built for.  NOTE: Anaconda Python not supported.
```

Comment:
nit, but I'm not sure we should mention all the languages/py implementations,... that we do not support. Rather saying we do support standard python 2.7, 3.2+ . 
There's also Cython, PyPy,... and I'm not sure if we'd build with that

Reply:
That is a good point.  Let me fix that.